specs/AppNotifications/AppNotifications-spec.md

```diff
@@ -139,6 +139,29 @@ int main()
 }
 ```
 
+## Registering for App Notifications using assets
+
+For Unpackaged applications, the developer calls into Register API with DisplayName and Icon.
+WinAppSDK will register and display them when an App Notification is received. The developer
+should provide both the assets or not provide them at all. If assets are not provided WinAppSDK
+will determine appropriate DisplayName and Icon. Icon provided by the developer should
+be valid and reside on the local machine only. For Packaged applications, this API is not
+applicable and will throw an exception.
+
+```cpp
+int main()
+{
+    auto manager = winrt::AppNotificationManager::Default();
+    manager.Register(L"AppNotifications", winrt::Windows::Foundation::Uri {L"<Path to Icon>"});
```

Comment:
I think it's okay to set a mock path (`C:\Users\<user>\AppData\MyApp\icon.png` or the app folder which may be under ProgramFiles?).

Reply:
Would like to see what everyone thinks of this before updating. Thanks.

Reply:
I like more specific over too general, as long as there aren't too many options and that we list the most common options. Maybe we use :\Users\<user>\AppData\MyApp\icon.png as proposed by Daniel and we also put a comment with an example path to a folder inside the program file folder. 

Maybe we should show the functions to call in order to retrieve the path(s) to the folders where the app is installed.

Reply:
Something like:

```c++
std::filesystem::path exePath{ GetPathToCurrentExe() };
auto iconPath{ exePath / "displayicon.png" };
```

Reply:
Sample should show use of MRT or another resource loader indirection to fetch that display string, too.